File: src/main/java/org/cimsbioko/activity/SyncDbActivity.java
package org.cimsbioko.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;
import androidx.appcompat.app.ActionBar;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;
import org.cimsbioko.R;


public class SyncDbActivity extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.sync_db_activity);
        Toolbar toolbar = findViewById(R.id.sync_db_toolbar);
        setSupportActionBar(toolbar);
        ActionBar actionBar = getSupportActionBar();
        if (actionBar != null) {
            actionBar.setDisplayHomeAsUpEnabled(true);
            actionBar.setHomeButtonEnabled(true);
        }
        setTitle(R.string.sync_db_title);
    }
}

File: src/main/kotlin/org/cimsbioko/activity/SyncDbActivity.kt
package org.cimsbioko.activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import org.cimsbioko.R
import org.cimsbioko.databinding.SyncDbActivityBinding

class SyncDbActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setTitle(R.string.sync_db_title)
        SyncDbActivityBinding.inflate(layoutInflater).apply {
            setContentView(root)
            setSupportActionBar(syncDbToolbar)
        }
        supportActionBar?.apply {
            setDisplayHomeAsUpEnabled(true)
            setHomeButtonEnabled(true)
        }
    }
}